packages/cli/src/options/beaconNodeOptions/network.ts

```diff
@@ -140,38 +132,6 @@ export const options: ICliCommandOptions<INetworkArgs> = {
     group: "network",
   },
 
-  "network.requestCountPeerLimit": {
```

Comment:
Expose the rate limit factor to allow disabling. Also note that removing this options from here will cause potential consumers from Lodestar starting at all. There's not reason to cause that here, I would mark them as deprecated until the next major release

Reply:
Also note that removing this options from here will cause potential consumers from Lodestar starting at all. There's not reason to cause that here, I would mark them as deprecated until the next major release